Instance,Runtime (seconds),Feasible,Profit,Error
lrc103C.txt,1.04,No,-inf,"[Error running script: Command '['python3', 'main.py', '--instance_path', '/home/akkermanfr/competition_master/Instances/lrc103C.txt']' returned non-zero exit status 1.. Details: Traceback (most recent call last):
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/main.py"", line 143, in <module>
    sol = main(instance)
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/main.py"", line 61, in main
    plot_routes(instance_dict,solutionNNh2,'output_NN_h2.png')
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/plot.py"", line 39, in plot_routes
    plt.savefig(f'Group19/Figures/{output_file}')
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/pyplot.py"", line 1134, in savefig
    res = fig.savefig(*args, **kwargs)  # type: ignore[func-returns-value]
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/figure.py"", line 3390, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py"", line 2193, in print_figure
    result = print_method(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py"", line 2043, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py"", line 497, in print_png
    self._print_pil(filename_or_obj, ""png"", pil_kwargs, metadata)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py"", line 446, in _print_pil
    mpl.image.imsave(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/image.py"", line 1656, in imsave
    image.save(fname, **pil_kwargs)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/PIL/Image.py"", line 2436, in save
    fp = builtins.open(filename, ""w+b"")
FileNotFoundError: [Errno 2] No such file or directory: 'Group19/Figures/output_NN_h2.png'
]"
lrc303C.txt,2.46,No,-inf,"[Error running script: Command '['python3', 'main.py', '--instance_path', '/home/akkermanfr/competition_master/Instances/lrc303C.txt']' returned non-zero exit status 1.. Details: Traceback (most recent call last):
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/main.py"", line 143, in <module>
    sol = main(instance)
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/main.py"", line 61, in main
    plot_routes(instance_dict,solutionNNh2,'output_NN_h2.png')
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/plot.py"", line 39, in plot_routes
    plt.savefig(f'Group19/Figures/{output_file}')
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/pyplot.py"", line 1134, in savefig
    res = fig.savefig(*args, **kwargs)  # type: ignore[func-returns-value]
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/figure.py"", line 3390, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py"", line 2193, in print_figure
    result = print_method(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py"", line 2043, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py"", line 497, in print_png
    self._print_pil(filename_or_obj, ""png"", pil_kwargs, metadata)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py"", line 446, in _print_pil
    mpl.image.imsave(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/image.py"", line 1656, in imsave
    image.save(fname, **pil_kwargs)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/PIL/Image.py"", line 2436, in save
    fp = builtins.open(filename, ""w+b"")
FileNotFoundError: [Errno 2] No such file or directory: 'Group19/Figures/output_NN_h2.png'
]"
lrc203C.txt,1.67,No,-inf,"[Error running script: Command '['python3', 'main.py', '--instance_path', '/home/akkermanfr/competition_master/Instances/lrc203C.txt']' returned non-zero exit status 1.. Details: Traceback (most recent call last):
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/main.py"", line 143, in <module>
    sol = main(instance)
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/main.py"", line 61, in main
    plot_routes(instance_dict,solutionNNh2,'output_NN_h2.png')
  File ""/home/akkermanfr/repos/tlm-assignment-2-furniture-distribution-company-submissions/tlm-assignment-2-furniture-distribution-company-group-19/Group19/plot.py"", line 39, in plot_routes
    plt.savefig(f'Group19/Figures/{output_file}')
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/pyplot.py"", line 1134, in savefig
    res = fig.savefig(*args, **kwargs)  # type: ignore[func-returns-value]
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/figure.py"", line 3390, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py"", line 2193, in print_figure
    result = print_method(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backend_bases.py"", line 2043, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py"", line 497, in print_png
    self._print_pil(filename_or_obj, ""png"", pil_kwargs, metadata)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/backends/backend_agg.py"", line 446, in _print_pil
    mpl.image.imsave(
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/matplotlib/image.py"", line 1656, in imsave
    image.save(fname, **pil_kwargs)
  File ""/home/akkermanfr/.local/lib/python3.10/site-packages/PIL/Image.py"", line 2436, in save
    fp = builtins.open(filename, ""w+b"")
FileNotFoundError: [Errno 2] No such file or directory: 'Group19/Figures/output_NN_h2.png'
]"
